node/provision.sh

```diff
@@ -53,6 +57,28 @@ nvm install 4.4.3
 NVM
 
 echo "---------------------------------------------"
+echo "------ Installing MongoDB database ----------"
+echo "---------------------------------------------"
+
+apt-get install -y mongodb-org
+
+echo "---------------------------------------------"
+echo "------ Installing Essential Packages --------"
+echo "---------------------------------------------"
+
+su - vagrant <<ESSENTIALS
+
+source ~/.nvm/nvm.sh
+echo "Installing bower"
+npm i -g bower
+echo "Installing gulp"
+npm i -g gulp
+echo "Installing Yeoman"
+npm i -g yo
+
+ESSENTIALS
+
+echo "---------------------------------------------"
 echo "--- Installing Workshopper modules ----------"
```

Comment:
No need for these I think. But if you can, check for the packages FCC usually recommends for back-end stuff.
`git-it`, `how-to-npm`, `express-works`, `mongo-workshopper` etc.